frontend/src/components/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import {
+  Button, 
+  CssBaseline,
+  Typography,
+  Container,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import StyledTextField from '../StyledTextField'
+
+export interface ResetPasswordProps {
+
+};
```

Comment:
seems unnecesary (unless you expect there might be some changes in the future here)

Reply:
I don't know how it will be done in the future, I just made a component by the way (looks similary to login and registration :P), someone else will take care of the logic, so I'll leave it as it is.